docs/content/en/docs/Cluster operator/quickstart.md

```diff
@@ -0,0 +1,121 @@
+---
+title: "Quick start"
+linkTitle: "Quick start"
+description: >
+  The easiest way to get start with Kurator.
+---
+
+In this tutorial we’ll cover the basics of how to use [Cluster API](https://cluster-api.sigs.k8s.io) and kurator cluster operator to create one or more Kubernetes clusters.
+
+## Prerequisites
+
+### Setup kubernetes clusters with Kind
+
+Deploy a kubernetes cluster using kurator's scripts. This script will create three clusters for you, one is used to host Karmada control plane and the other two will be joined as member clusters.
+
+```bash
+git clone https://github.com/kurator-dev/kurator.git
+cd kurator
+hack/local-setup-cluster.sh
+```
+
+### Install cert manager
+
+Kurator cluster operator depends on [cert manager CA injector](https://cert-manager.io/docs/concepts/ca-injector).
+
+***Please make sure cert manager is ready before install cluster operator***
+
+```console
+helm repo add jetstack https://charts.jetstack.io
+helm repo update
+kubectl create namespace cert-manager
+helm install -n cert-manager cert-manager jetstack/cert-manager --set installCRDs=true
+```
+
+## Build `cluster-operator` from source
+
+Build cluster operator image:
+
+```console
+make docker
+```
+
+Build cluster operator helm chart:
+
+```console
+make gen-chart
+```
+
+Load image to kind cluster:
+
+```console
+kind load docker-image ghcr.io/kurator-dev/cluster-operator:latest --name kurator
+```
+
+## Install cluster operator
+
+```console
+# Please make sure cert manager is ready before install cluster operator
+kubectl create namespace kurator-system
+helm install kurator-base out/charts/base-0.1.0.tgz -n kurator-system
+helm install -f <your_custom_values>.yaml -n kurator-system kurator-cluster-operator out/charts/cluster-operator-0.1.0.tgz
+```
+
+## Create a vanilla cluster on AWS
+
+The clusterctl CLI tool handles the lifecycle of a Cluster API management cluster.
+
+```console
+# download clusterctl
+curl -L https://github.com/kubernetes-sigs/cluster-api/releases/download/v1.2.5/clusterctl-linux-amd64 -o clusterctl
+sudo install -o root -g root -m 0755 clusterctl /usr/local/bin/clusterctl
+# verify version
+clusterctl version
+```
+
+Create a kubernetes cluster on AWS, which contains control plane nodes and worker nodes, the cluster topology shows as follows:
+
+{{< image width="75%"
+    link="./image/capa-aws.png"
+    >}}
+
+```console
+kubectl apply -f manifests/examples/capi-quickstart.yaml
+
+# verify the control plane is up 
+kubectl get kubeadmcontrolplane -w
+```
+
+***The control plane won’t be Ready until we install a CNI in the next step.***
```

Comment:
what's the error result?

Reply:
there's no error result, `kubeadmcontrolplane` won't be ready.